### PR TITLE
Add StreamPi Server

### LINKED ISSUE #20317: Stream-Pi Server
# Package Requested

- [x] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [ ] I have performed a search and couldn't find this package.
- [ ] I think there is a new version available and I have provided the URL.

Manifest:

Fill out as much of the manifest metadata you can:

```YAML
PackageIdentifier:   Stream_Pi.Server
PackageVersion:      EA+3
PackageLocale:       en-US
Publisher:           Stream-Pi
PackageName:         Stream-Pi Server
License:             GPL-3.0
ShortDescription:    The Early Access Release of Stream-Pi Server (Ver. EA+3)
Installers: 
 - Architecture:     x64
   InstallerType:    msi
   InstallerUrl:     https://github.com/stream-pi/server/releases/download/1.0.0-EA%2B3/stream-pi-server-windows-x64-1.0.0-EA+3-installer.msi
   InstallerSha256:  0c5f6fa3665ca14290f01621746bdb9a599277fdd34e31d69d3ea617f78d6af3
ManifestType: singleton
ManifestVersion: 1.0.0
```

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
Fixes #20317 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20405)

### CODE REVIEW

File: manifests/s/Stream-Pi/Server/1.0/Stream-Pi.Server.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+#  Scope: 
+  InstallerUrl: https://github.com/stream-pi/server/releases/download/1.0.0-EA%2B3/stream-pi-server-windows-x64-1.0.0-EA+3-installer.msi
+  InstallerSha256: 0C5F6FA3665CA14290F01621746BDB9A599277FDD34E31D69D3EA617F78D6AF3
+#  ProductCode:
```
Comment (quhxl) [inline manifests/s/Stream-Pi/Server/1.0/Stream-Pi.Server.installer.yaml:18]:
```suggestion
  ProductCode: '{E6636B37-5A72-303B-B698-990585EDBE8D}'
```

File: manifests/s/Stream-Pi/Server/1.0/Stream-Pi.Server.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+#  Scope:
```
Comment (quhxl) [inline manifests/s/Stream-Pi/Server/1.0/Stream-Pi.Server.installer.yaml:15]:
```suggestion
  Scope: machine
```

**Review by quhxl — CHANGES_REQUESTED** [review]

File: manifests/s/Stream-Pi/Server/1.0/Stream-Pi.Server.installer.yaml (hunk 2 above)
Comment (OfficialEsco) [reply manifests/s/Stream-Pi/Server/1.0/Stream-Pi.Server.installer.yaml:15]: The Scope should be mandatory tbh, as voted here https://github.com/microsoft/winget-pkgs/discussions/16219#discussioncomment-827007

**Review by ItzLevvie — APPROVED** [review]
image

**Pull request merged**